.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add an ignore property for any IDE files such as this.
Here's a tutorial about it:
https://help.github.com/articles/ignoring-files/